templates/card.php

```diff
@@ -0,0 +1,47 @@
+<!-- COMPONENT:START -->
+<div class="wsu-c-card__wrapper">
+	<div class="wsu-c-card__container">
+		<div class="wsu-c-card__content">
+			<?php if ( in_array('photo', $display_fields) && !empty($photo) ) : ?>
+				<figure class="wsu-c-card__photo-frame">
+					<img class="wsu-c-card__photo" src="<?php echo $photo; ?>" alt="Photo of <?php echo esc_attr( $name ); ?>" data-object-fit>
+				</figure>
+			<?php endif; ?>
+
+			<?php if ( in_array('name', $display_fields) && !empty($name) ) : ?>
+				<?php echo $opening_heading_tag . esc_html( $name ) . $closing_heading_tag; ?>
```

Comment:
$opening_heading_tag should be escaped here (not earlier). Same for $closing_heading_tag